pkg/resources/management.cattle.io/v3/cluster/Cluster.md

```diff
@@ -0,0 +1,5 @@
+## Cluster
+
+### Annotations validation
+
+When a cluster is created if `field.cattle.io/creator-principal-name` annotation is set then `field.cattle.io/creatorId` annotation must be set as well and the user's principal name in the former should match the creator's user principal id.
```

Comment:
These are two separate checks so they should be different sentences for clarity
```suggestion
When a cluster is created and `field.cattle.io/creator-principal-name` annotation is set then `field.cattle.io/creatorId` annotation must be set as well. 
The value of `field.cattle.io/creator-principal-name` should match the creator's user principal id.
```

Reply:
Nice rewording!